torch/fx/symbolic_trace.py

```diff
@@ -392,8 +392,8 @@ def _unpatch_wrapped_functions(orig_fns : List[PatchedFn]):
 def wrap(fn_or_name : Union[str, Callable]):
     """
     This function can be called at global scope in a module to cause
-    references to the global function secified by `fn_name` to use
-    them in FX.
+    references to the global function specified by `fn_or_name` to use
```

Comment:
I think this line is sort of unclear. In particular, I'm not sure what "global" means or "use them in FX" means.

Maybe something like:
This function can be called at module-level scope to register `fn_or_name` as a "leaf function". A "leaf function" will be preserved as a CallFunction node in the FX trace instead of being traced through